gitlabform/configuration/core.py

```diff
@@ -162,13 +163,64 @@ def is_group_skipped(self, group) -> bool:
     def is_skipped(self, an_array: list, item: str) -> bool:
         pass
 
+    @staticmethod
+    def validate_break_inheritance_flag(config, **kwargs):
+        has_invalid_flag = False
+        level = None
+        for key, value in config.items():
+            if "inherit" == key:
+                if kwargs["parent"] == "empty" and (
+                    kwargs["level"] == "group"
+                    or kwargs["level"] == "project"
+                    or kwargs["level"] == "subgroup"
```

Comment:
Nit: syntax `kwargs["level"] in ["group", "project", "subgroup"]` would be shorter and more readable.

Reply:
After the changes, this does not exist anymore.